vsintegration/tests/FSharp.Editor.IntegrationTests/InProcess/WorkspaceInProcess.cs

```diff
@@ -0,0 +1,16 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Microsoft.VisualStudio.Extensibility.Testing;
+
+internal partial class WorkspaceInProcess
+{
+    public async Task WaitForAsyncOperationsAsync(CancellationToken cancellationToken)
```

Comment:
I am confused, as to why this is needed and `WaitForProjectSystemAsync` cannot simply be used in its place

Reply:
Yeah right there was more code there initially but now it doesn't make much sense, refactores.